services/frontend-service/src/ui/ConfirmationDialog.tsx

```diff
@@ -156,7 +156,7 @@ export const ConfirmationDialogProvider = (props: ConfirmationDialogProviderProp
             <AlertTitle>Warning: Not deployed to "{undeployedUpstream}" yet!</AlertTitle>
             {[
                 `This version is not yet deployed to "${undeployedUpstream}" environment.`,
-                'Your changes will be overridden by the next release train.',
+                'Your changes may be overridden by the next release train.',
```

Comment:
I believe this shouldn’t be fixed here, we are trying to keep releases PRs clean and only for releasing

Reply:
removed